Improve batch changes site admin guide (#25350)
sourcegraph/sourcegraph: doc/batch_changes/references/faq.md

- The changeset was imported (and was therefore already published by someone or something else).

The changeset may also be in a state that we cannot currently publish from: for example, because a previous push to the code host failed (in which case you should re-apply the batch change), or if you are actively detaching the changeset from your batch change.

> ### Changesets are taking a lot of time to sync

Reviewer: For consistency:

```suggestion
### Why do my changesets take a long time to sync?
```
